Tests/Client/YooChooseNotifierTest.php

```diff
@@ -250,14 +250,10 @@ protected function getContentTypeServiceMock($contentTypeId)
      */
     protected function getRepositoryServiceMock($identifier)
     {
-        $repositoryInterfaceServiceMock = $this->getMock('eZ\Publish\API\Repository\Repository');
-        $signalDispatcherServiceMock = $this->getMock('eZ\Publish\Core\SignalSlot\SignalDispatcher');
-
-        $repositoryServiceMock = $this->getMock(
-            'eZ\Publish\Core\SignalSlot\Repository',
-            array('sudo', 'getContentService', 'getContentTypeService'),
-            array($repositoryInterfaceServiceMock, $signalDispatcherServiceMock)
-        );
+        $repositoryServiceMock = $this
+            ->getMockBuilder('\eZ\Publish\Core\SignalSlot\Repository')
```

Comment:
Ideally I would drop PHP `5.4` (and `5.5`) testing to use my favourite `::class` here, but with this bundle on 1.2 branch I'm not sure if I can.

Reply:
We can't, EZP 5.4 supports php 5.4